vendor/bat-native-ads/src/bat/ads/internal/bundle.cc

```diff
@@ -45,8 +47,8 @@ bool Bundle::UpdateFromCatalog(const Catalog& catalog) {
 
   auto callback = std::bind(&Bundle::OnStateSaved,
       this, bundle_state->catalog_id, bundle_state->catalog_version,
-      bundle_state->catalog_ping,
-      bundle_state->catalog_last_updated_timestamp_in_seconds, _1);
+          bundle_state->catalog_ping,
+              bundle_state->catalog_last_updated_timestamp_in_seconds, _1);
```

Comment:
Alignment - and the next change below.

Reply:
Alignment is correct as 4 spaces if following on from the previous line